conf/application.conf

```diff
@@ -120,6 +120,11 @@ microservice {
   }
 
   services {
+    artefact-processor {
+      host = "localhost"
+      port = 9000
```

Comment:
does ap run on 9000?
looking at sm --ports that might be in use already?
9000 -> CUSTOMS_MANAGE_AUTHORITIES_FRONTEND

Reply:
we don't currently run on service-manager (so haven't defined a port) - it needs queuing to trigger a request to artefact-processor. So this can be revisited whenever we need to run on service-manager.